getting-started/revo-to-full-manual.txt

```diff
@@ -1,122 +1,217 @@
-    • For most combat situations and scenarios, Revolution is OK enough.
+*Note: a **Table of Contents** can be found in the pinned messages.*
 
-⬥ Full manual is a combat mode where the game does not activate any abilities for you. Every ability you want to use in combat will require an input from the player.
-    • With no inputs from the player, full manual will only use auto attacks.
-    • Full manual offers more control over what abilities are used, when they are used, and how they are used.
+Runescape has 3 different combat modes: **revolution (revo)**, **revolution++ (revo++)** and **full manual (FM)**.
+This guide aims to cover what these different game modes are, what a player just starting out might want to use, and how to transition to using full manual.
 
-.
-> __**Why You Should Switch to Full Manual**__
-.tag:comparison
-**__A Comparison Between Revolution and Full Manual__**
-
-⬥ **__Channeled abilities__**
-    • Usually last longer than a Global Cooldown (3 ticks or 1.8s).
-    • Have an extra delay between the last hit of the ability and the start of the next ability.
-        - This generally only applies to Snipe <:snipe:535541258425204770>, (Greater) Concentrated Blast <:gconc:869285393223254107> / <:conc:535533833106489365>, and Fury <:fury:535532879510372352>
-        - This delay can be negated by manually inputting the next ability.
-        - Revolution will let the delay run its course, which wastes time and damage.
+Combat modes can be configured by going to **Esc > Gameplay Settings > Combat & Action Bar > Combat Mode**.
 
-⬥ Below is a comparison of manually canceling a channeled ability vs just letting revo do it for you. Notice how the player with revolution enabled suffers a 1-tick delay on Greater Concentrated Blast <:gconc:869285393223254107>
 .
-https://i.imgur.com/9C2JEZQ.mp4
+> __**Related Channels**__
+.tag:related
+For further information and reading, you can refer to the following resources.
 .
-*Note: Manually canceling channeled abilities can be done on Revolution mode. But, it still requires a manual input from the player with precise timing.*
+{
+  "embed": {
+    "title": "__Quick Navigation Links__",
+    "description": "Links to related channels and resources for more info about PvM and DPS.\nUse the **Help Channels** if you want to ask any questions, or post clips/videos for review and feedback.",
+    "color": 39423,
+    "fields": [
+      {
+        "name": "__Getting Started__",
+        "value": "⬥ <#1020546518941454438>\n⬥ <#1020546905865986068>\n⬥ <#1020547206303985714>",
+        "inline": true
+      },
+      {
+        "name": "__DPM-Advice Channels__",
+        "value": "⬥ <#1020729755064074291>\n⬥ <#1020729582086782996>\n⬥ <#1020729671127662643>",
+        "inline": true
+      },
+      {
+        "name": "__Help Channels__",
+        "value": "⬥ <#656898197561802760>\n⬥ <#1019648841965916220>",
```

Comment:
Should include pvm help, pvm help forums, and also VOD review (not sure which these 2 are, just noticed there were 2 not 3)